qcut/docs/task/e2e-results-2026-04-29/results.md

```
@@ -0,0 +1,77 @@
+pr: 293
+runner: bun run test:e2e:bg
+duration: 11.7m
+total: 142
+passed: 118
+skipped: 24
+failed: 0
+---
+
+# E2E run — 2026-04-29 (branch `win-Hermes`, PR #293)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)